src/main/index.ts

```diff
@@ -1,16 +1,26 @@
 import { app, BrowserWindow } from 'electron';
+import configureStore from "../shared/configureStore";
+import {exampleAliasedAction} from "../shared/actions/example";
 declare const MAIN_WINDOW_WEBPACK_ENTRY: any;
 
+
 // Handle creating/removing shortcuts on Windows when installing/uninstalling.
 if (require('electron-squirrel-startup')) {
   app.quit();
 }
 
+const store = configureStore('main');
+
 const createWindow = (): void => {
+  store.subscribe(() => console.log('action received in main'))
```

Comment:
This is temporary -> will be deleted. It logs the text in the console (where we run `npm start`)